app/src/main/java/edu/uc/app/parkfinder/dto/Park.kt

```diff
@@ -2,6 +2,9 @@ package edu.uc.app.parkfinder.dto
 
 import com.google.gson.annotations.SerializedName
```

Comment:
All updates to this file were coding styles according to: https://developer.android.com/kotlin/style-guide